Libraries/LibURL/URL.cpp

```diff
@@ -430,7 +430,7 @@ bool code_point_is_in_percent_encode_set(u32 code_point, PercentEncodeSet set)
     case PercentEncodeSet::Path:
         return code_point_is_in_percent_encode_set(code_point, PercentEncodeSet::Query) || "?^`{}"sv.contains(static_cast<char>(code_point));
     case PercentEncodeSet::Userinfo:
-        return code_point_is_in_percent_encode_set(code_point, PercentEncodeSet::Path) || "/:;=@[\\]^|"sv.contains(static_cast<char>(code_point));
+        return code_point_is_in_percent_encode_set(code_point, PercentEncodeSet::Path) || "/:;=@[\\]|"sv.contains(static_cast<char>(code_point));
```

Comment:
Could you combine both changes into a single commit? They are so obviously related that if we ever would want to revert it, both changes should be reverted at the same time.

Reply:
done